bundles/net.tourbook.export/src/net/tourbook/export/ExportImages.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (C) 2023 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.export;
+
+/**
+ * Contains image constants which do not need to be translated
+ */
+public class ExportImages {
+
+   public static final String Export_FIT_Logo        = "fit-logo.png";        //$NON-NLS-1$
+   public static final String Export_FIT_Logo_Dark   = "fit-logo-dark.png";   //$NON-NLS-1$
+   public static final String Export_Garmin_Logo     = "garmin-logo.png";     //$NON-NLS-1$
+   public static final String Export_GPX_Logo        = "gpx-logo.png";        //$NON-NLS-1$
+   public static final String Export_MyTourbook_Logo = "mytourbook-logo.png"; //$NON-NLS-1$
```

Comment:
In the dark theme the GPX and FIT icons looks pixelated and spongy, and I cannot recognize what is displayed. In the light theme it looks better.

![image](https://user-images.githubusercontent.com/1283445/236382006-76f935c3-1a64-4c8d-b746-2d464140f5db.png)


The export and print icons could also be larger like the upload icons

![image](https://user-images.githubusercontent.com/1283445/236385311-bd008e7a-6fef-42a0-a10d-d6f35baed522.png)

Reply:
>In the dark theme the GPX and FIT icons looks pixelated and spongy, and I cannot recognize what is displayed. In the light theme it looks better.

I will try to enlarge them to see if it looks better. I have looked and don't see that the vendors offer icons for "dark themes"

Reply:
I think the black binoculars in the GPX icon will not look much better, it's a black symbol with dark background

Reply:
Fixed (enlarged the icons)

I changed the `GPX` icon with a home-made icon that is well-displayed when in dark mode

![image](https://user-images.githubusercontent.com/25235665/236923288-d048bab0-86f9-4c74-b9fb-46c0be84aa35.png)
![image](https://user-images.githubusercontent.com/25235665/236923303-5b845f00-7019-47c7-99e4-d5dbeb118adf.png)